natlib/svcd.c

```diff
@@ -185,3 +185,68 @@ static int svcd_init( lua_State *L )
 
     return 0;
 }
+
```

Comment:
You need to register your function at the beginning of this file under SVCD_SYMBOLS. There is a template for the init function already so you just need to follow it.